.gitignore

```diff
@@ -21,6 +21,7 @@ native/*/Makefile
 native/*/autom4te.cache/
 native/*/config.status
 native/*/configure
+native/*/configure~
```

Comment:
I don't suppose this is created by our project? This seems to be created by user tools. Adding system-specific files to the repo's .gitignore is considered a poor practice

Reply:
It's created when `make` flavour is used (e.g., in our Dockerfile [here](https://github.com/modcluster/mod_proxy_cluster/blob/2cbc93876f4f0ba688527a311f74aa766bf2f682/test/httpd/Dockerfile#L33)). I see it on par with the `build` folder for `cmake`.

Reply:
I see, never mind then!